script/jenkins/branches/Jenkinsfile

```
@@ -23,6 +23,8 @@ pipeline {
             sh '''#!/bin/bash
             export PATH=$PATH:$CONDAPATH
             source activate petals_env
+            mamba install xesmf -c conda-forge -y
+            pip install cdsapi dantro matplotlib>3.4 rioxarray ruamel.yaml
```

Try upgrading to ensure matplotlib>3.4 is installed
```suggestion
            pip install --upgrade cdsapi dantro matplotlib>3.4 rioxarray ruamel.yaml
```